doc/content/guides/getting-started/console/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Console"
+description: ""
+weight: 8
+---
+
+<style>
+img {
+  border: 1px solid #ddd;
+}
+</style>
+
+## Login
+
+You need to be logged in inside the Console in order to create gateways, applications, devices and API keys. With the stack running in one terminal session, open https://localhost:8885 in your browser.
+
+>Note: If you are hosting your stack on a remote machine, you should replace `https://localhost:8885` with your own host address and port.
```

Comment:
Also http://localhost:1885.

TBH, I'm not not sure if this adds much

Reply:
Yeah, those who can deploy on a remote machine should know that already.